packages/manager/modules/observability-to-customer/src/components/timeControls/DateTimePicker.component.tsx

```diff
@@ -0,0 +1,43 @@
+import {
+  Datepicker,
+  DatepickerContent,
+  DatepickerControl,
+  FormField,
+  FormFieldLabel,
+  Timepicker,
+  TimepickerControl,
+} from '@ovhcloud/ods-react';
+
+import { formatTimeFromDate } from '@/utils/dateTimeUtils';
+
+interface DateTimePickerProps {
+  id: string;
+  label: string;
+  defaultValue?: number;
+}
+
+export const DateTimePicker: React.FC<Readonly<DateTimePickerProps>> = ({
+  id,
+  label,
+  defaultValue,
+}): JSX.Element => {
+  const defaultDate = defaultValue ? new Date(defaultValue) : new Date();
+  const defaultTime = formatTimeFromDate(defaultDate);
```

Comment:
Could be managed directly inside the function as
```suggestion
   const defaultTime = formatTimeFromDate(defaultDate);
```
```ts
formatTimeFromDate(sDate: string) {
const date = defaultValue ? new Date(sDate) : new Date();
//...
// your logic of formatting
// ...
}
```

Reply:
defaultDate is used in another part of the component (not just for computing defaultTime)